lib/puppet/provider/aem_installer/default.rb

```diff
@@ -21,7 +21,7 @@ def initialize(resource = nil)
     @stop_file = 'stop'
     @launchpad_name = 'cq-quickstart-*-standalone*.jar'
     @repository_dir = 'repository'
-    @quickstart_fields = [:home, :version]
+    @quickstart_fields = %i(home version)
```

Comment:
You can revert this. i realize that the rubocop is failing. I won't take that into consideration; but i will need to run a suite of acceptance tests before i merge.